lassen/alu.py

```diff
@@ -206,13 +207,16 @@ def __call__(self, alu: ALU_t, signed_: Signed_t, a: Data, b: Data, d:Bit) -> (D
                 if (abs_input[15] == Bit(1)):
                     scale = SInt[16](15)
                 normmant_mul_left = SInt[16](abs_input)
-                normmant_mul_right = (SInt[16](1) << (SInt[16](15)-scale))
+                #normmant_mul_right = (SInt[16](1) << (SInt[16](15)-scale))
+                normmant_mul_right = (SInt[16](15)-scale)
                 normmant_mask = SInt[16](0x7f00)
 
             #if (alu == ALU_t.FCnvInt2F) | (alu == ALU_t.FCnvExp2F):
             if (scale >= 0):
+                print("scale ")
```

Comment:
Please no extraneous print statements. Also this is likely the cause of the travis failure